View/index.html

```diff
@@ -47,6 +47,12 @@
         <div class="trash"><img src="../assets/trash.png"></div>
         <div class="trash"><img src="../assets/trash.png"></div>
         
+        <div class="lives-container">
+            <div id="heart-1" class="heart"><img src="../assets/heart.png"></div>
```

Comment:
Three lines are ok in my opinion (others wouldn't be so lenient). But what if tomorrow I asked you to add one more life?
Try to write more flixible code. For instance above, you changed the three to a variable.